.editorconfig

```diff
@@ -0,0 +1,40 @@
+## EditorConfig is awesome: http://EditorConfig.org
+root = true
+
+
+## Newlines section
+
+# Make sure all files use lf as line ending
+[*]
+end_of_line = lf
```

Comment:
I think it's dangerous to do this, since you can have for example imports with legacy data that can contain other end of line characters. You don't want these import files to change when you enter them in your editor. I'd specify this end_of_line rule for all code formats, but not every file format available.